src/cmakeProject.ts

```diff
@@ -2472,6 +2473,36 @@ export class CMakeProject implements api.CMakeToolsAPI {
     async selectEnvironments() {
         return null;
     }
+
+    async getExpansionOptions(): Promise<ExpansionOptions> {
+        const workspaceFolder: string = this.workspaceContext.folder.uri.fsPath;
+        return {
+            vars: {
+                buildKit: this.activeKit?.name || "",
```

Comment:
Please follow the pattern of the other ExpansionOptions creation function by replacing unsupported properties with themselves instead of empty string.

```ts
  vars: {
    buildKit: this.activeKit?.name || "${buildKit}",
    buildKitVendor: "${buildKitVendor}",
    ...
  }
```